### ISSUE TITLE
ENH: Taxonium Electron CLI

### ISSUE THREAD

**Issue by corneliusroemer** [theosanderson/taxonium#428]
Alright this is a low priority ergonomic proposal: allow opening of taxonium electron from the CLI with passing of the file.

I don't like to use my mouse to find files ;) 

Would be cool if I could do `taxonium {{filename}}` or something like this.

Should be doable in principle.

**Comment by theosanderson** [comment]
Yep the idea of allowing you to drag onto the unopened Taxonium icon (which I think is the same idea under the hood) is definitely on the radar.

**Comment by corneliusroemer** [comment]
Unopened taxonium icon? Not sure that was my idea :)

In the other issue I was thinking of dropping it anywhere in the window, not just on the button "choose file".

<img width="763" alt="image" src="https://user-images.githubusercontent.com/25161793/195964319-7bb14f6d-5b6c-4946-a4bb-c723869acdaa.png">

CLI may be a bit different.

**Comment by theosanderson** [comment]
Yep, different to #423, but nonetheless something I intended todo. And I think under the hood that is implemented as recognising `Taxonium.app file.xyz` and loading file.xyz
